src/environment.py

```diff
@@ -3,26 +3,30 @@
 from pygame.surface import Surface
 from typing import Tuple
 
+from src.player import Player
+
 
 class Environment:
     def __init__(
-        self, screen: Surface, agent_size: int = 10, render_on: bool = False
+        self, screen: Surface, player_size: int = 10, render_on: bool = False
```

Comment:
In `main.py`, should line 12 (`env = Environment(screen=screen, render_on=True)`) be updated with the player size as well?